notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,65 @@
+# Integration of payment into checkout process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Deployment](#deployment)
+  - [Glossary](#glossary)
+  - [Requirements](#requirements)
+- [Configuration](#configuration)
+      - [Environment variables to configure the notification module:](#environment-variables-to-configure-the-notification-module)
+  - [After deployment](#after-deployment)
+    - [Register the endpoint](#register-the-endpoint)
+- [Note](#note)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Deployment
+
+### Glossary
+In this process, there are 2 parties involved:
+
+**Adyen** - the payment provider which will send notifications
+**Notification module** - hosted service that receives notifications from Adyen,
+processes and stores them on a commercetools platform project.
+
+### Requirements 
```

Comment:
for consistency, I would prefer to use `Prerequisites` instead of `Requirements` with the same format with the DevelopmentGuide.md

Reply:
Changed everything to Requirements
https://github.com/commercetools/commercetools-adyen-integration/commit/54826250bf63027bdf4a753e6be18568a6313b2c#diff-7d9966222b5102e10716ec93fe9b5a2aR13